contracts/core/EntryPoint.sol

```diff
@@ -345,11 +346,13 @@ contract EntryPoint is IEntryPoint, StakeManager {
     /**
      * Get counterfactual sender address.
      *  Calculate the sender contract address that will be generated by the initCode and salt in the UserOperation.
-     * this method always revert, and returns the address in SenderAddressResult error
+     * this method always revert, and returns the address (and the nonce) in SenderAddressResult error
      * @param initCode the constructor code to be passed into the UserOperation.
      */
     function getSenderAddress(bytes calldata initCode) public {
```

Comment:
Maybe rename it to `getSenderNonce` then?

Reply:
not relevant anymore, since this nonce was removed.